.github/comment-templates/comment-pending-deployment-info.md

```diff
@@ -0,0 +1,9 @@
+Hey @{{ .community_rep }} and @{{ .issue_author }}! 👋
+I noticed there is still pending information about the new hub deployment.
+Can you please help us fill it in?
+
+The information pieces still missing, are: **{{ .pending_info }}**
```

Comment:
could this be formatted as a list or a little table? It might be easier to parse that way.

Reply:
Is something like this good enough @choldgraf?

I had to format the `pending_info` string because I don't think there's a way to parse a list or a string from within this kind of markdown template and https://github.com/chuhlomin/render-template won't know what to do with it.

<img width="796" alt="Screenshot 2023-01-31 at 18 09 14" src="https://user-images.githubusercontent.com/7579677/215815659-ac7144ed-94bf-4634-9e20-012c1fbc5c1c.png">